src/style.css

```diff
@@ -1,3 +1,91 @@
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --primary-color: #000;
+  --hover-color: #00acdc;
+  --header-background-color: #fff;
+}
+
+html {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  margin: 0 auto;
+  background-color: var(--header-background-color);
+  padding: 0 50px;
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+}
+
+.logo .logo-image {
+  width: 40px;
+}
+
+.nav {
+  margin: 0;
+}
+
+.nav__list {
+  margin: 0;
+  display: flex;
+  list-style: none;
+  padding: 0;
+}
+
+.nav__item {
+  margin-right: 20px;
```

Comment:
Be consistent with vertical margins. It's recommended to add either top or bottom margin but not both to avoid issues with margin collapsing. In this case, you might want to consider using only bottom margins for '.nav__item'.